includes/class-wc-stripe-helper.php

```
@@ -802,6 +802,83 @@ public static function add_stripe_methods_in_woocommerce_gateway_order( $ordered
+		foreach ( array_keys( $gateway_order ) as $gateway_id ) {
+			// Skip internal WooCommerce Payments entries.
+			if ( 0 === strpos( $gateway_id, '_wc_pes_' ) ) {
+				continue;
+			}
+
+			// The first non-internal gateway decides position one.
+			return str_contains( $gateway_id, 'stripe' );
+		}
+
+		return false;
+	}
```

PHPMD (2.15.0)

[error] The method is_stripe_in_position_one_in_woocommerce_gateway_order is not named in camelCase. (undefined)

(CamelCaseMethodName)